src/cli.rs

```diff
@@ -27,11 +27,9 @@ impl Cli {
         let mut source = String::new();
         self.input.read_to_string(&mut source)?;
         match self.format {
-            Dialect::Sql => writeln!(
-                self.output,
-                "TODO! do something useful with source:\n{}",
-                &source
-            )?,
+            Dialect::Ast => self
```

Comment:
Oh clever, I had been thinking of it as a different subcommand but this is better!